src/App.js

```diff
@@ -9,7 +11,7 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <Article title={article.title} text={article.text} date={article.date} />
```

Comment:
Три аргумента поместились, но это неудобно поддерживать если я добавлю четвертый
Лучше заранее перенести каждый на новую строку, если их два или больше
```suggestion
  <Article
    title={article.title}
    text={article.text}
    date={article.date}
  />
```